src/eda_server/db/models/job.py

```diff
@@ -35,6 +36,26 @@
     sa.Column("uuid", postgresql.UUID),
 )
 
+
+activation_job_instances = sa.Table(
```

Comment:
Is it simply a many-to-many junction table?

Reply:
I think it is one-to-many, activation can have multiple job instances but each job instance only belongs to one activation.

Reply:
So why the extra table is needed then?